apps/token/src/i18n/translations/dev.json

```diff
@@ -199,6 +199,7 @@
   "governanceRequired": "Required",
   "daysLeft": "{{daysLeft}} left to vote.",
   "toVote": "to vote",
+  "youNeed{{amount}}AssociatedVegaTokensToVote": "You need a minimum of {{amount}} associated $VEGA tokens to vote",
```

Comment:
chore: this isn't how other strings with varaibles are defined, would be nice to have consistency